migrations/1753214286445_txs-status-index.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined;
+
+exports.up = pgm => {
+  pgm.createIndex('txs', 'status', {
```

Comment:
Adding the index is great but I don't think it will help a lot especially for the `success` status, given that the vast majority of transactions are successful.

I'm worried that running the endpoint on mainnet will cause performance issues when it tries to calculate the `COUNT(*)` of records to show in the `total`. We'll need to run load test on this.